lib/config.js

```diff
@@ -32,11 +32,23 @@ function expandTilde(apath) {
     return path.join(homedir, apath.slice(2));
```

Comment:
I think a cleaner solution than doing something like this would be to just check it process.env.CONFIG_FILE_PATH is defined. If it is, use that. Otherwise check for the command line parameter. Else use the default config location. I don't see the need to do all this resolving using regex and all of that.